src/post/post.repository.knex.ts

```diff
@@ -379,6 +393,31 @@ export class KnexPostRepository {
                         (accountId) => accountId,
                     );
                 }
+            } else if (isDeletedPost) {
+                const existingRow = await transaction('posts')
+                    .select('deleted_at')
+                    .where({
+                        id: post.id,
+                    })
+                    .first();
+
+                if (existingRow && existingRow.deleted_at === null) {
+                    await transaction('posts')
+                        .update({
+                            deleted_at: transaction.raw('CURRENT_TIMESTAMP'),
```

Comment:
Based on what was discussed in Slack, do we want to use `UTC_TIMESTAMP` here now, or do we want to do a big bang change later?

Reply:
Yeah so I wanted to do a big bang change so that we can do any migrations if necessary and all the current data is consistent